src/fakeredis_instance.erl

```diff
@@ -132,6 +136,7 @@ parse_data(Data, #state{parser_state = ParserState} = State) ->
 -spec handle_data(Command :: list(), #state{}) -> #state{}.
 handle_data([Cmd | Args] = Data, State) ->
     CmdUpper = string:uppercase(Cmd),
+    maybe_kill_node(CmdUpper, State),
```

Comment:
maybe you idea here was to use Data instead of first command element only?

Reply:
Yes, my idea was to match the whole command including parameters. It can even be case sensitive, since it's our own special feature (as long as it's documented).

It could be implemented as a separate clause of handle_data/2 before this one, because we're not continuing after the krash/kill anyway.

```Erlang
handle_data(Command, #state{kill_on_command = Command, local_port = Port}) ->
    kill_node(Port);
...
```

I don't mind that we match only the command without parameters, though, if you think it's better. This makes it more similar to how the delay parameter. But it doesn't really matter. (The kill feature is per node anyway and delay is for all nodes...)